assets/js/common/Table/Table.test.jsx

```diff
@@ -139,6 +139,16 @@ describe('Table component', () => {
       .forEach((tableRow) => expect(tableRow).toHaveClass(pxClass));
   });
 
+  it('should have wrapper div classes with rounded and overflow hidden', () => {
+    const data = tableDataFactory.buildList(10);
+
+    render(<Table config={tableConfig} data={data} />);
+
+    const wrapperDiv = screen.getByRole('table').closest('div');
+
+    expect(wrapperDiv).toHaveClass(/rounded/, 'overflow-hidden');
```

Comment:
@janvhs, so I'm not too sure how to propose a change in the test here 😅... the condition that the test is checking for is the existence of both the `rounded*` and `overflow-hidden` classes which are required for the rounded corner effect. I could just check for the `overflow*`class but it won't yield the same result in the UI unless it is specially the `overflow-hidden` class. Please let me know if this what you were referring to.

Reply:
Ah yes, I agree. I did raise it in the Test Section that the test might be too much work for such a little change. But at the same time I felt like it just helpful to try use a test to try safeguard a Design System aesthetic. 😅

Reply:
Yeah we also have visual tests in our pipeline because Chromatic does them, I just disabled them because they cost additional money and due to the nature of our stories using fixtures the always changing data was conflicting with snapshot tests.

Feel free to merge, @jagabomb!

Reply:
@janvhs thanks for the suggestion... I will definitely look into it for next time.